docs/variants/novacustom_nv4x_adl/releases.md

```diff
@@ -1,20 +1,114 @@
+- [Keyboard backlight not working after restart](https://github.com/Dasharo/dasharo-issues/issues/349)
+- [Warning sign in device manager under the touchpad device](https://github.com/Dasharo/dasharo-issues/issues/194)
+- [Popup with information about recovery mode is displayed after flashing with a valid binary](https://github.com/Dasharo/dasharo-issues/issues/269)
+- [The power LED is not always blinking while in sleep mode on Windows 11](https://github.com/Dasharo/dasharo-issues/issues/182)
+- [Internal keyboard stops working randomly in firmware](https://github.com/Dasharo/dasharo-issues/issues/295)
+- [DTS on network boot fails to boot](https://github.com/Dasharo/dasharo-issues/issues/502)
+- [Firmware Update Mode does not automatically boot into DTS](https://github.com/Dasharo/dasharo-issues/issues/506)
+- [Permanent keyboard illumination after cold boot fails: keyboard light level is not saved correctly](https://github.com/Dasharo/dasharo-issues/issues/501)
+- [Controlling keyboard illumination (brightess and on/off) does not work](https://github.com/Dasharo/dasharo-issues/issues/498)
+- [Permanent keyboard illumination not working in Ubuntu 22.04](https://github.com/Dasharo/dasharo-issues/issues/517)
+- [Controlling brightness and turning off RGB keyboard illumination does not work (Ubuntu 22.04)](https://github.com/Dasharo/dasharo-issues/issues/514)
+- [Wi-Fi card is not recognized on Windows 11](https://github.com/Dasharo/dasharo-issues/issues/512)
+- [No HDMI audio on Windows 11](https://github.com/Dasharo/dasharo-issues/issues/511)
+- [No HDMI display output on Windows 11](https://github.com/Dasharo/dasharo-issues/issues/510)
+- [No USB-C display output on Windows 11](https://github.com/Dasharo/dasharo-issues/issues/509)
+- [Keyboard backlight is disabled after suspend (Windows 11)](https://github.com/Dasharo/dasharo-issues/issues/507)
+- [Reset by F9 sets battery thresholds to 0](https://github.com/Dasharo/dasharo-issues/issues/499)
+
+### Known issues
+
+- [Connecting and immediately disconnecting the charger, sets the battery status in OS to 'charging' for about 2 minutes](https://github.com/Dasharo/dasharo-issues/issues/350)
+- [Power state after power failure option does not work as intended](https://github.com/Dasharo/dasharo-issues/issues/524)
+- [No ability to change active PCR banks with TPM PPI in FW](https://github.com/Dasharo/dasharo-issues/issues/521)
+
+### Binaries
+
+[novacustom_nv4x_adl_ec_v1.7.0.rom][novacustom_nv4x_adl_ec_v1.7.0.rom_file]{.md-button}
+[sha256][novacustom_nv4x_adl_ec_v1.7.0.rom_hash]{.md-button}
+[sha256.sig][novacustom_nv4x_adl_ec_v1.7.0.rom_sig]{.md-button}
+
+[novacustom_nv4x_adl_v1.7.0.rom][novacustom_nv4x_adl_v1.7.0.rom_file]{.md-button}
+[sha256][novacustom_nv4x_adl_v1.7.0.rom_hash]{.md-button}
+[sha256.sig][novacustom_nv4x_adl_v1.7.0.rom_sig]{.md-button}
+
+[novacustom_nv4x_adl_v1.7.0_dev_signed.rom][novacustom_nv4x_adl_v1.7.0_dev_signed.rom_file]{.md-button}
```

Comment:
We need to check why `dev_signed` binaries were generated for NV4X ADL only with our release script.s